### ISSUE TITLE
Instance: The "@" (mention) feature constitutes an improperly implemented combobox/listbox pattern | WCAG 2.2 Violation - There are improperly implemented custom combobox/listbox elements present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#32]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1.
